clusterdock/cli.py

```diff
@@ -103,6 +103,10 @@ def main():
     start_parser.add_argument('-c', '--cluster-name',
                               help='Cluster name to use',
                               metavar='name')
+    start_parser.add_argument('--clusterdock-config-directory',
+                              help='Directory where files generated by clusterdock are available',
+                              default=defaults['DEFAULT_CLUSTERDOCK_CONFIG_DIRECTORY'],
+                              metavar='confdir')
```

Comment:
to be consistent with other places, change `'confdir'` -> `'path'`

Reply:
Done.